master/buildbot/steps/master.py

```diff
@@ -49,6 +49,7 @@ def __init__(self, command, **kwargs):
         self.usePTY = kwargs.pop('usePTY', 0)
         self.interruptSignal = kwargs.pop('interruptSignal', 'KILL')
         self.logEnviron = kwargs.pop('logEnviron', True)
+        self.runtimeTimeout = kwargs.pop('runtimeTimeout', 3600)
```

Comment:
Let's use snake_case for new APIs. This is documented somewhere that buildbot is moving away from camelCase to standard python formatting.

Reply:
No problem, I hesitated 🙈 done in https://github.com/buildbot/buildbot/compare/b71c7e3913463904876c179a2ba4cce20faabac8..eef8b14817c8dcdb412017299916d6671cc370b7

Reply:
... and https://github.com/buildbot/buildbot/compare/eef8b14817c8dcdb412017299916d6671cc370b7..882128dfe80769ad0efac0c8b0d5fc7c4ecb518b 🤦